### PR TITLE
Coverage Increased for owasp/management/commands

### LINKED ISSUE #284: Increase backend owasp/management coverage
**Is your feature request related to a problem? Please describe.**
We don't have any coverage here and want it to be 75%+

```
apps/owasp/management/commands/owasp_aggregate_projects.py                         70     70     16      0     0%   3-102
apps/owasp/management/commands/owasp_enrich_chapters.py                            30     30      8      0     0%   3-52
apps/owasp/management/commands/owasp_enrich_committees.py                          30     30      4      0     0%   3-55
apps/owasp/management/commands/owasp_enrich_projects.py                            30     30      4      0     0%   3-53
apps/owasp/management/commands/owasp_scrape_chapters.py                            40     40     10      0     0%   3-76
apps/owasp/management/commands/owasp_scrape_committees.py                          40     40     10      0     0%   3-76
apps/owasp/management/commands/owasp_scrape_projects.py                            52     52     12      0     0%   3-88
```

### PR DESCRIPTION
**Pull request by Naveen-Pal**
Resolves #284 

Added tests for owasp/management/commands/*
<br>
Below are the snapshot of the coverage percentage.
image
image

### CODE REVIEW

File: backend/tests/owasp/management/commands/owasp_aggregate_projects_tests.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        mock_repository.contributors_count = 5
+        mock_repository.forks_count = 2
+        mock_repository.open_issues_count = 4
+        mock_repository.releases.count.return_value = 1
+        mock_repository.stars_count = 50
+        mock_repository.subscribers_count = 3
+        mock_repository.watchers_count = 7
+        mock_repository.top_languages = {"Python", "JavaScript"}
+        mock_repository.license = "MIT"
+        mock_repository.topics = {"security", "owasp"}
```
Comment (arkid15r) [inline backend/tests/owasp/management/commands/owasp_aggregate_projects_tests.py]: Technically those are lists

File: backend/tests/owasp/management/commands/owasp_aggregate_projects_tests.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        mock_active_projects.__iter__.return_value = iter(mock_projects_list)
+        mock_active_projects.count.return_value = len(mock_projects_list)
+        mock_active_projects.__getitem__.side_effect = (
+            lambda idx: mock_projects_list[idx.start : idx.stop]
+            if isinstance(idx, slice)
+            else mock_projects_list[idx]
+        )
+        mock_active_projects.order_by.return_value = mock_active_projects
+
+        with mock.patch.object(Project, "active_projects", mock_active_projects), mock.patch(
```
Comment (arkid15r) [inline backend/tests/owasp/management/commands/owasp_aggregate_projects_tests.py]: Please format mocks with `()`

File: backend/tests/owasp/management/commands/owasp_enrich_chapters_tests.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        mock_active_chapters.__iter__.return_value = iter(mock_chapters_list)
+        mock_active_chapters.count.return_value = len(mock_chapters_list)
+        mock_active_chapters.__getitem__.side_effect = (
+            lambda idx: mock_chapters_list[idx.start : idx.stop]
+            if isinstance(idx, slice)
+            else mock_chapters_list[idx]
+        )
+        mock_active_chapters.without_geo_data.order_by.return_value = mock_active_chapters
+
+        with mock.patch.object(
```
Comment (arkid15r) [inline backend/tests/owasp/management/commands/owasp_enrich_chapters_tests.py]: Make it more readable with `()`

**Review by arkid15r — CHANGES_REQUESTED** [review]
Looks good, 
please format/group mocks in all files:

**Comment by Naveen-Pal** [comment]
@arkid15r I have updated the PR. Let me know if any other updates required.

**Review by arkid15r — APPROVED** [review]
LGTM 👍

**Pull request merged**